sovtoken/sovtoken/token_req_handler.py

```diff
@@ -150,28 +150,32 @@ def _reqToTxn(self, req: Request):
     def _update_state_mint_public_txn(self, txn, is_committed=False):
         payload = get_payload_data(txn)
         seq_no = get_seq_no(txn)
-        for output in payload[OUTPUTS]:
-            self._add_new_output(Output(output["address"], seq_no, output["amount"]),
-                                 is_committed=is_committed)
+        [self._add_new_output(Output(output["address"], seq_no,
+                                     output["amount"]), is_committed=is_committed) for output in payload[OUTPUTS]]
 
     def _update_state_xfer_public(self, txn, is_committed=False):
         payload = get_payload_data(txn)
-        for inp in payload[INPUTS]:
-            self._spend_input(inp["address"], inp["seqNo"], is_committed=is_committed)
-        for output in payload[OUTPUTS]:
-            seq_no = get_seq_no(txn)
-            self._add_new_output(Output(output["address"], seq_no, output["amount"]),
-                                 is_committed=is_committed)
+        seq_no = get_seq_no(txn)
+
+        [self._spend_input(inp["address"], inp["seqNo"],
+                           is_committed=is_committed) for inp in payload[INPUTS]]
+
+        [self._add_new_output(Output(output["address"], seq_no, output["amount"]),
+                              is_committed=is_committed) for output in payload[OUTPUTS]]
```

Comment:
I know this is how it was previously, but there are constants for "address", "seqNo", and "amount" in `sovtoken/constants.py`.

Reply:
I agree